keps/sig-scheduling/5075-dra-consumable-capacity/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA: Consumable Capacity
+kep-number: 5075
+authors:
+  - "@sunya-ch"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+  - sig-network
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-30-01
+reviewers:
+  - "@johnbelamaric"
+  - "@catblade"
+approvers:
+  - TBD
```

Comment:
Either @dom4ha or @sanposhiho could be listed here.

@dom4ha, are you okay with taking this?

Reply:
I have added. Thank you!